src/components/common/contactsMessengerPicker/ContactsMessengerPicker.tsx

```diff
@@ -14,13 +14,13 @@ import { Language } from 'src/types/language';
 
 type Props = {
   readonly input: {
-    readonly error: FormikErrors<string>;
-    readonly touched: FormikTouched<string>;
+    readonly error?: FormikErrors<string>;
+    readonly touched?: FormikTouched<string>;
```

Comment:
Since you are using the `readonly` keyword within nested types, you can create a utility type to handle this pattern. This is a common use case, and you can find many examples of such utilities with a quick search on Google. (For [example](https://dev.to/bwca/deep-readonly-generic-in-typescript-4b04))
And if you do not deed nesting you can use native typescript [helper](https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype)

Reply:
Thanks!

Reply:
https://github.com/boarlabsxyz/Kavoon/pull/57/commits/8c8d9ded63b5d2ded0d733a4821b136b06769957